frontend/src/lib/components/flows/content/FlowModule.svelte

```diff
@@ -99,11 +99,22 @@
 		width,
 		threshold: FLOW_MODULE_WIDTH_THRESHOLD
 	})
+
+	afterUpdate(() => {
```

Comment:
This entire logic boils down to the height of the EditorBar height, which is a constant. 
We can directly fetch its height using a svelte binding on the component to avoid this afterUpdate logic.

Reply:
There might be more than one element above the `Splitpane` and we need to get the sum of the height of those elements. I've tried the way you mentioned but it really didn't work. I also think this is a lot cleaner because I didn't need to modify thos components just to get their height.